bookkeeper-server/pom.xml

```diff
@@ -178,8 +208,26 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>${netty.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
+      <version>${netty-tcnative.version}</version>
+      <classifier>${os.detected.classifier}</classifier>
```

Comment:
For me the issue mostly is making sure that we document clearly when a release (binary or otherwise) is arch specific, which features depend on that arch specific code, and if there is a simple way to work around it (like rebuild on the platform you want to run on) how to do that.

Hard coding the architecture to x86_64 does not feel ideal to me.